index.pug

```diff
@@ -377,6 +391,64 @@ html(lang='en')
                       | .
                     p.text-muted
                       | Find a detailed billing forecast and history on your project.
+    
+    section#process.process
+      .container
+        .row
+          .col-md-12
+            h2 What is the software localization process
+            p Software-localization is the process of transforming the language of a web app, game or any web content to the end user's locale. A translation management platform helps tech companies and developers to increase the confidence for new potential customers, as localized products are best for local markets. The primary benefit of locize is #[strong to simplify the core process of localization] with a very easy to use translation management platform. Starting with just one piece of code the translations can be managed with an easy interface and can be deloyed separately with no delays.
+
+            img(style='max-width: 100%;', src='img/localization_process.png', alt='localization-process')
```

Comment:
maybe split the image "localization_process.png" to html + 3 images, to be able to localize to other languages

Reply:
the strategy with this image was to rank on a "google image pack". This is only possible as one image. I can create a german version of the image. Is this possible to integrate various images for languages?

Reply:
ahh, ok.
Let's keep it like this for now.

Thx for the explanation.